testsuite/integration-arquillian/HOW-TO-RUN.md

```diff
@@ -85,6 +85,21 @@ TODO: Add info about Wildfly logging
     mvn -f testsuite/integration-arquillian/tests/base/pom.xml \
         -Dtest=org.keycloak.testsuite.adapter.**.*Test
 
+### Jetty
+
+At the moment we can run the testsuite with Jetty `9.1`, `9.2` and `9.4`. 
+Each version has its corresponding profile:
+
+* Jetty `9.1`: `app-server-jetty91`
+* Jetty `9.2`: `app-server-jetty92`
+* Jetty `9.4`: `app-server-jetty94`
+
+Here's how to run the tests with Jetty `9.4`:
+
+    mvn -f testsuite/integration-arquillian/tests/base/pom.xml \
```

Comment:
@slaskawi This is probably wrong. I think you need to either add another step in which you build integration-arquillian/servers/app-server/jetty or change this path to:

````
mvn -f testsuite/integration-arquillian/pom.xml
````

Reply:
You're right. Yet I believe this minor issue can be fixed later

Reply:
Yes, it also requires a server module to be built. Probably I could change the path to `testsuite/integration-arquillian`. @mhajas Would that work for you? And would you guys like me to add it now? @mhajas @hmlnarik

Reply:
With @hmlnarik we decided not to do it in this PR. It is just a minor thing which can be fixed in my PR (Tomcat) so that we won't need to wait for another CI runs for this PR.

Reply:
Ok, great! Thanks guys!